packages/react-components/src/components/ChatMessage/MessageComponents/FluentChatMessageComponent.tsx

```diff
@@ -48,22 +48,20 @@ export const FluentChatMessageComponent = (props: FluentChatMessageComponentWrap
     onDisplayDateTimeString,
     inlineImageOptions,
     /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-    fileDownloadHandler,
+    actionForAttachment,
     userId,
     /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-    onRenderFileDownloads,
+    onRenderAttachmentDownloads,
```

Comment:
Should this only be behind the `attachment-download` CC flag?
Is there a further PR to separate these concerns?

Reply:
i am planning to leave CC refactor out of this PR since @jimchou-dev  is working on it